src/content/docs/tutorial-app-performance/fixing-high-latency.mdx

```diff
@@ -5,82 +5,92 @@ tags:
 - Customer experience
 - Browser monitoring
 - Latency
-metaDescription: "How to find, diagnose, and fix, an instance of your abnormal or poor app latency."
+metaDescription: "Diagnose and resolve latency in your apps so your users get the most out of their site experience."
 redirects:
   - /docs/journey-performance/fixing-high-latency/
 ---
 
 import journeyAbnormalLCP from 'images/browser_screenshot-crop_LCP-abnormal.webp'
 
+Diagnosing and resolving performance issues helps your organization attract new users while retaining existing ones. [According to Google's research](https://think.storage.googleapis.com/images/micromoments-guide-to-winning-shift-to-mobile-download.pdf), 29% of mobile users will avoid using a site or an application if it's too slow or too difficult to find information. Of that 29% of users, 70% leave because of site latency. 
 
-Your app's performance is one of the most important aspects of whether or not your business will succeed. [According to Google's research](https://think.storage.googleapis.com/images/micromoments-guide-to-winning-shift-to-mobile-download.pdf), 29% of mobile users will avoid using a site or an application if it's too slow or too difficult to find information, with 70% of that user group doing so because the site is too slow. Diagnosing and fixing performance issues for your site or application helps your organization attract new users while retaining existing ones, and New Relic is just the tool to help you do it.
+While core web vitals capture how assets behave on your page from a user perspective, they're one part of resolving poor app performance. 
```

Comment:
This looks a little lonely out here, what do you think about adding it to the preceding mini-paragraph or adding some context?